### ISSUE TITLE
Idea

### ISSUE THREAD

**Issue by vincenzopalazzo** [material-ui-swing/material-ui-swing-donations#1]
Hi @atarw

I'm giving a course on web programming at university and I wanted to start seeing some technology and I thought that maybe a donation site for the material-ui-swing project would suit us.

First of all, I don't want to bother you and if you don't like the idea, tell me clearly so I delete the project.

Instead, if you like feeling free to add and remove what you want, the notification of collaboration should have arrived, I already made some changes I hope you like them.

I await your news and excuse my bad English

**Comment by atarw** [comment]
Yeah you've really helped me a lot with this project so if someone is generous enough then I don't see anything wrong with it :)

**Comment by vincenzopalazzo** [comment]
@atarw Since you are quite busy, I can try to develop a page for you so that it becomes a reference site for donations, what do you say?

If you would also like me to work with you on this project, I would be grateful if you joined me, so it's easier for me to keep the two versions aligned, what do you say?

**Comment by atarw** [comment]
Yeah sure that would be great, thank you for your help with this!!

**Comment by atarw** [comment]
I kind of want to keep the projects separate just because you have additional non-swing components, but in terms of the donations I have no issue with splitting it equally.

**Comment by vincenzopalazzo** [comment]
@atarw Absolutely, I wanted to collaborate with your project because it makes no sense to work on two similar projects that change in a few things, for me it would be better to focus only on one job and prefer to do it with you because working in two is better than in one.

My non-swing components can become additional modules and the user can decide whether or not to import them into his project.
My components have been implemented to live one's life even without look and feel.

Let me know if you like to join me as a contributor :

**Comment by vincenzopalazzo** [comment]
Hi @atarw,
I finished the first version of the site, in the page folder, there is the template with which I created my bio that you can easily copy if you don't want to waste time or you simply don't have time.

When you add the references, tell us so we update our readme :)

Good evening
